lib/web_ui/lib/src/engine/keyboard.dart

```diff
@@ -22,17 +22,17 @@ class Keyboard {
   }
 
   /// The [Keyboard] singleton.
-  static Keyboard get instance => _instance;
-  static Keyboard _instance;
+  static Keyboard? get instance => _instance;
+  static Keyboard? _instance;
```

Comment:
Since this is initialized in:
static void initialize() {
    _instance ??= Keyboard._();
  }

late?